web-ui/src/main/resources/catalog/components/viewer/partials/mainviewer.html

```diff
@@ -30,6 +30,12 @@
       <span class="label" ng-show="showToolLabels" translate="">Layers</span>
       <span role="tooltip" translate="">Layers</span>
     </button>
+    <button class="btn btn-default" ng-model="activeTools.projectionSwitcher"
```

Comment:
To be honest I'm not sure this is the best place to put the projection switcher widget (I mean with the map tools): I think this could simply be a plain dropdown somewhere on the side of the map. There are already many map tools, and finding an icon that actually conveys the purpose of the tool is very hard. I suggest we place this somewhere else (I'm sure @MichelGabriel  will have good ideas for this!)

Reply:
Yes, there are many tools, but it is possible to hide some map tools in the settings. 

The icons are always difficult, I can't really find one in the font awesome set. For map-related icons we could also have a look at: http://mapsk.in/

I made a quick mockup of the suggestion made by @jahow (see image). I like the idea, but it requires some rebuilding/restyling.

![gn-switch-projections](https://user-images.githubusercontent.com/19608667/45745773-e05d4c00-bc01-11e8-8768-1412de75a6a6.png)

If the switcher stays like it is it would be nice to have some extra info/explanation for the user, like with the filter tool.

Reply:
I can change it to be like this smaller dropdown/selection, but then I can't add more info/explanation. What would be the best for usability? Maybe a Tooltip or an info button? Did we do that in other parts of GN where I can copy how to do it?

About the icons, I think that should be an independent PR, I don't want to add more features here. Don't you think so @jahow ?

Reply:
You could use a tooltip with a bit more information (like 'change the projection of the map'). The tooltips are also used for the other tools in the map, so with a tooltip it will look the same as the rest of the elements on the map.

Reply:
Honestly I think a dropdown containing projection codes is almost enough to understand what it does, especially if the user can just try and see for him/herself. A tooltip would definitely be enough.

BTW I think Michel's mockup looks great. :+1: